docs/faststore/docs/headless-cms/multiple-page-template.mdx

```diff
@@ -73,7 +73,9 @@ N --> O{Yes} --> P(Use matching template)
 N --> Q{No} --> R{Use generic template}
 ```
 
-The system first searches for a template named after the product slug. For example, if the product slug is `/blue-shirts`, FastStore will look for a template named `/blue-shirts`.
+The system first searches for a template named after the product slug. For example, if the product slug is `/blue-shirts/p`, FastStore will look for a template named `/blue-shirts/p`.
+
+> ⚠️ Keep in mind that if a more specific slug template with the product sku id is found, such as `/blue-shirts-99988212/p`, exists, it will take precedence when the slug matches the product slug without the sku id `/blue-shirts/p`.
```

Comment:
Minha sugestão rápida seria:
```suggestion
> ⚠️ If an SKU template is available, it takes precedence over the product template. For example, if a template is available for slugs containing the SKU ID (e.g., `/blue-shirts-99988212/p`), this template will take precedence over a more generic product template for `/blue-shirts/p`.
```

Mas acho que o ideal seria atualizar o diagrama e a lista que segue...

> The system first searches for a template named after the SKU slug. For example, if the SKU slug is `/blue-shirts-99988212/p`, FastStore will look for a template named  ???.
>
> 1. If no template matches the specified SKU, FastStore searches for a template based on the product slug. For example, if there's no template available for `/blue-shirts-99988212/p`, FastStore will look for a more generic product template for `/blue-shirts/p`.

@Mariana-Caetano

Reply:
looks good to me :)